.github/workflows/main.yml

```diff
@@ -1,5 +1,9 @@
 name: Build Dalamud
-on: [push, pull_request, workflow_dispatch]
+on:
+  push:
+  pull_request:
+    types: [opened]
```

Comment:
This will cause subsequent builds to not trigger for PRs. This should not be the case and the default types for `pull_request` is `opened`, `syncronize`, `reopened` which all handle different cases with pushes to the pull request for commits and if a pull request is opened again after closing. Best is to revert this change as you want it to build for each time something happens to the commit tree for the PR.

Reply:
Hmm, when testing on my fork the `push` trigger was firing whenever I made a commit to my draft PR. That could've been something on my end - I'll revert this portion though just to be safe.